README.md

```diff
@@ -58,6 +58,18 @@ rosdep install serial
 # Install python dependencies
 pip3 install wheel  # firstly install the wheel for further package building if you start from a empty virtual environment.
 pip3 install -r src/marimbabot/requirements.txt
+
+# installation of precise-runner for keyword spooting
+# first, make sure all the necessary software are installed.
+sudo apt install python3-pip \
```

Comment:
Please remove this because it is installed via the package.xml / rosdep right now.

Reply:
sure